var/spack/repos/builtin/packages/intel-oneapi-compilers/package.py

```diff
@@ -334,6 +350,12 @@ def install(self, spec, prefix):
         ftn = find("fortran-installer", "*")
         if ftn:
             self.install_component(ftn[0])
+        # install nvidia-plugin
+        if self.spec.satisfies("+nvidia"):
+            nvidia_script = find("nvidia-plugin-installer", "*")
+            if nvidia_script:
+                self.install_component(nvidia_script[0],is_nvidia=True)
```

Comment:
It might be good to have an error message or warning if +nvidia but there is no plugin for that version.

Reply:
I'd opt for a `conflicts("@version-range +nvidia)` instead of throwing an error.

Reply:
using the conflicts is a good way to do it.